0.41.0rc1 fails to build on FreeBSD 13.2: AttributeError: Undefined symbol "LLVMPY_LLJITLookup"

## Reporting a bug

```
===================================================================================== test session starts =====================================================================================
platform freebsd13 -- Python 3.9.17, pytest-7.4.0, pluggy-1.2.0 -- /usr/local/bin/python3.9
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/usr/ports/devel/py-llvmlite/work-py39/llvmlite-0.41.0rc1/.hypothesis/examples')
Using --randomly-seed=3325686616
rootdir: /usr/ports/devel/py-llvmlite/work-py39/llvmlite-0.41.0rc1
plugins: anyio-3.7.1, forked-1.6.0, hypothesis-6.82.2, datadir-1.4.1, cov-2.9.0, randomly-3.12.0, timeout-2.1.0, time-machine-2.11.0, rerunfailures-11.1.2, flaky-3.7.0, typeguard-4.1.2, xdist-2.5.0, env-0.6.2, mock-3.10.0
collected 5 items / 4 errors                                                                                                                                                                  

=========================================================================================== ERRORS ============================================================================================
_____________________________________________________________________ ERROR collecting conda-recipes/llvmlite/run_test.py _____________________________________________________________________
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:531: in collect
    self._inject_setup_module_fixture()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:545: in _inject_setup_module_fixture
    self.obj, ("setUpModule", "setup_module")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:310: in obj
    self._obj = obj = self._getobj()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:528: in _getobj
    return self._importtestmodule()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/local/lib/python3.9/site-packages/_pytest/pathlib.py:565: in import_path
    importlib.import_module(module_name)
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
/usr/local/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:178: in exec_module
    exec(co, module.__dict__)
conda-recipes/llvmlite/run_test.py:6: in <module>
    main()
llvmlite/tests/__init__.py:56: in main
    res = run_tests()
llvmlite/tests/__init__.py:50: in run_tests
    prog = unittest.main(suite=suite, testRunner=runner, exit=False,
llvmlite/tests/customize.py:38: in __init__
    super(NumbaTestProgram, self).__init__(*args, **kwargs)
/usr/local/lib/python3.9/unittest/main.py:100: in __init__
    self.parseArgs(argv)
llvmlite/tests/customize.py:72: in parseArgs
    super(NumbaTestProgram, self).parseArgs(argv)
/usr/local/lib/python3.9/unittest/main.py:133: in parseArgs
    self._main_parser.parse_args(argv[1:], self)
/usr/local/lib/python3.9/argparse.py:1828: in parse_args
    self.error(msg % ' '.join(argv))
/usr/local/lib/python3.9/argparse.py:2582: in error
    self.exit(2, _('%(prog)s: error: %(message)s\n') % args)
/usr/local/lib/python3.9/argparse.py:2569: in exit
    _sys.exit(status)
E   SystemExit: 2
--------------------------------------------------------------------------------------- Captured stderr ---------------------------------------------------------------------------------------
usage: __main__.py [-h] [-v] [-q] [--locals] [-f] [-c] [-b]
                   [-k TESTNAMEPATTERNS] [-R] [-m] [--profile]
                   [tests ...]
__main__.py: error: unrecognized arguments: -rs -o
_______________________________________________________________________ ERROR collecting llvmlite/tests/test_binding.py _______________________________________________________________________
llvmlite/binding/ffi.py:134: in __getattr__
    return self._fntab[name]
E   KeyError: 'LLVMPY_LLJITLookup'

During handling of the above exception, another exception occurred:
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:531: in collect
    self._inject_setup_module_fixture()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:545: in _inject_setup_module_fixture
    self.obj, ("setUpModule", "setup_module")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:310: in obj
    self._obj = obj = self._getobj()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:528: in _getobj
    return self._importtestmodule()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/local/lib/python3.9/site-packages/_pytest/pathlib.py:565: in import_path
    importlib.import_module(module_name)
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
/usr/local/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:178: in exec_module
    exec(co, module.__dict__)
llvmlite/tests/test_binding.py:17: in <module>
    from llvmlite import binding as llvm
llvmlite/binding/__init__.py:17: in <module>
    from .orcjit import *
llvmlite/binding/orcjit.py:298: in <module>
    ffi.lib.LLVMPY_LLJITLookup.argtypes = [
llvmlite/binding/ffi.py:137: in __getattr__
    cfn = getattr(self._lib, name)
/usr/local/lib/python3.9/ctypes/__init__.py:387: in __getattr__
    func = self.__getitem__(name)
/usr/local/lib/python3.9/ctypes/__init__.py:392: in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
E   AttributeError: Undefined symbol "LLVMPY_LLJITLookup"
_________________________________________________________________________ ERROR collecting llvmlite/tests/test_ir.py __________________________________________________________________________
llvmlite/binding/ffi.py:134: in __getattr__
    return self._fntab[name]
E   KeyError: 'LLVMPY_LLJITLookup'

During handling of the above exception, another exception occurred:
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:531: in collect
    self._inject_setup_module_fixture()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:545: in _inject_setup_module_fixture
    self.obj, ("setUpModule", "setup_module")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:310: in obj
    self._obj = obj = self._getobj()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:528: in _getobj
    return self._importtestmodule()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/local/lib/python3.9/site-packages/_pytest/pathlib.py:565: in import_path
    importlib.import_module(module_name)
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
/usr/local/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:178: in exec_module
    exec(co, module.__dict__)
llvmlite/tests/test_ir.py:14: in <module>
    from llvmlite import binding as llvm
llvmlite/binding/__init__.py:17: in <module>
    from .orcjit import *
llvmlite/binding/orcjit.py:298: in <module>
    ffi.lib.LLVMPY_LLJITLookup.argtypes = [
llvmlite/binding/ffi.py:137: in __getattr__
    cfn = getattr(self._lib, name)
/usr/local/lib/python3.9/ctypes/__init__.py:387: in __getattr__
    func = self.__getitem__(name)
/usr/local/lib/python3.9/ctypes/__init__.py:392: in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
E   AttributeError: Undefined symbol "LLVMPY_LLJITLookup"
______________________________________________________________________ ERROR collecting llvmlite/tests/test_refprune.py _______________________________________________________________________
llvmlite/binding/ffi.py:134: in __getattr__
    return self._fntab[name]
E   KeyError: 'LLVMPY_LLJITLookup'

During handling of the above exception, another exception occurred:
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/usr/local/lib/python3.9/site-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:531: in collect
    self._inject_setup_module_fixture()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:545: in _inject_setup_module_fixture
    self.obj, ("setUpModule", "setup_module")
/usr/local/lib/python3.9/site-packages/_pytest/python.py:310: in obj
    self._obj = obj = self._getobj()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:528: in _getobj
    return self._importtestmodule()
/usr/local/lib/python3.9/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/local/lib/python3.9/site-packages/_pytest/pathlib.py:565: in import_path
    importlib.import_module(module_name)
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
/usr/local/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:178: in exec_module
    exec(co, module.__dict__)
llvmlite/tests/test_refprune.py:3: in <module>
    from llvmlite import binding as llvm
llvmlite/binding/__init__.py:17: in <module>
    from .orcjit import *
llvmlite/binding/orcjit.py:298: in <module>
    ffi.lib.LLVMPY_LLJITLookup.argtypes = [
llvmlite/binding/ffi.py:137: in __getattr__
    cfn = getattr(self._lib, name)
/usr/local/lib/python3.9/ctypes/__init__.py:387: in __getattr__
    func = self.__getitem__(name)
/usr/local/lib/python3.9/ctypes/__init__.py:392: in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
E   AttributeError: Undefined symbol "LLVMPY_LLJITLookup"
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 4 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
====================================================================================== 4 errors in 2.43s ======================================================================================
*** Error code 2
```

- [x] I have tried using the latest released version of llvmlite (most recent is
 visible in the change log (https://github.com/numba/llvmlite/blob/main/CHANGE_LOG).
- [ ] I have included a self contained code sample to reproduce the problem.
  i.e. it's possible to run as 'python bug.py'.

--- a/ffi/Makefile.freebsd
+++ b/ffi/Makefile.freebsd
@@ -11,7 +11,7 @@ LIBS = $(LLVM_LIBS)
 INCLUDE = core.h
 SRC = assembly.cpp bitcode.cpp core.cpp initfini.cpp module.cpp value.cpp \
 	executionengine.cpp transforms.cpp passmanagers.cpp targets.cpp dylib.cpp \
-	linker.cpp object_file.cpp
+	linker.cpp object_file.cpp orcjit.cpp
 OUTPUT = libllvmlite.so
 
 all: $(OUTPUT)

